Functions/BIDS/xASL_bids_determineImageTypeGE.m

```diff
@@ -40,14 +40,26 @@
     if length(jsonPar.ImageType) == 5 && strcmpi(jsonPar.ImageType{1},'DERIVED') && strcmpi(jsonPar.ImageType{2},'PRIMARY') &&...
```

Comment:
Some of these typical names might be vendor-independent, if it helps to know this in the future.

Reply:
In the future yes. Currently, they are pretty vendor specific.

Reply:
OK can we then make a Philips section (and for other vendors)? That shouldn't take too much time, and does help code readability a lot

Reply:
Yes. but we don't have yet sections for other, because we can't read that for others. We have issues for that. So best we make sections for Philips and Siemens once we have them.